node/task.ts

```diff
@@ -14,6 +14,7 @@ require('./extensions');
 
 export enum TaskResult {
     Succeeded = 0,
+    SucceededWithIssues = 2,
```

Comment:
that line will always pass zero (otherwise will transfer control to the catch block, which uses the enum)